src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1087,6 +1093,64 @@ public Inventory get(final InventoryClickEvent e) {
 				return e.getClickedInventory();
 			}
 		}, 0);
+		// InventoryDragEvent
+		EventValues.registerEventValue(InventoryDragEvent.class, Player.class, new Getter<Player, InventoryDragEvent>() {
+			@Override
+			public @Nullable Player get(InventoryDragEvent event) {
+				return event.getWhoClicked() instanceof Player ? (Player) event.getWhoClicked() : null;
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(InventoryDragEvent.class, World.class, new Getter<World, InventoryDragEvent>() {
```

Comment:
no need to register the world event value here, it'll automatically do this from grabbing the player

Reply:
No it doesn't